composer.json

```diff
@@ -6,5 +6,19 @@
         "wp-coding-standards/wpcs": "^0.14.0"
     },
     "require": {
+    },
+    "extra": {
+        "hooks": {
+            "pre-commit": [
+            "echo committing as $(git config user.name)",
+            "sh bin/block-commits-with-merge-conflict.sh"
+            ]
+        }
+    },
+    "scripts": {
+        "post-install-cmd": "vendor/bin/cghooks add --ignore-lock",
```

Comment:
For these scripts to work we will also need package `brainmaestro/composer-git-hooks` check our [Astra's composer.json](https://github.com/brainstormforce/astra/blob/master/composer.json) for it's implementation example.

While you are at it, update version use the same packages from Astra's composer config and add `name` and `description` in the composer.json to make the syntax valid.

Reply:
Updated composer.json